src/components/FileTreeNode/index.tsx

```diff
@@ -114,13 +118,21 @@ export class FileTreeNodeBase<TreeNodeType> extends React.Component<Props> {
     let linterType = null;
     if (linterMessages && hasLinterMessages) {
       linterType = findMostSevereTypeForPath(linterMessages, node.id);
+      let title = getTitleForType(linterType, isFolder);
+
+      if (
+        linterMessages[node.id] &&
+        linterMessages[node.id].global.some(
+          ({ message }) => message === LINTER_MESSAGE_KNOWN_LIBRARY,
+        )
+      ) {
+        linterType = 'known-library';
```

Comment:
In the case that `linterType` is already defined, I don't think it should be overridden. Consider this case: jquery was detected but the version of jquery they used has been banned. That should show up as a red error since it would lead to rejection.

<img width="1151" alt="Screenshot 2019-03-19 16 37 08" src="https://user-images.githubusercontent.com/55398/54643647-6a795680-4a65-11e9-9ab3-c06ab1b1393c.png">

Reply:
Good call, thanks